internal/terraform/node_resource_partial_plan.go

```diff
@@ -301,6 +301,9 @@ func (n *nodeExpandPlannableResource) knownModuleSubgraph(ctx EvalContext, addr
 		DynamicTransformer(func(graph *Graph) error {
 			// We'll add nodes for any orphaned resources.
 			rs := state.Resource(addr)
+			if rs == nil {
```

Comment:
Nice, I ran into this one as well. I went for skipping this transformer on ephemeral nodes since it was a bit less confusing to the reader, but this is ok with me as well.

Reply:
yeah, I figured you would have hit this too, for now it was the minimal change to run the tests ;)